docs/pages/debugging/tools.mdx

```diff
@@ -191,6 +191,13 @@ To start debugging:
 
 This will attach VS Code to your running app.
 
+Alternatively, if you want a fully-featured IDE setup in VS Code, you might want to check out [Radon IDE](https://ide.swmansion.com/) (paid). This extension turns your editor into a powerful environment designed specifically for React Native and Expo projects, with advanced debugging, a network inspector, router integration, and other built-in tools. There's a 30-day free trial, so you can explore everything it offers before committing.
```

Comment:
```suggestion
Alternatively, if you want a fully-featured IDE setup in VS Code, you might want to check out the [Radon IDE](https://ide.swmansion.com/) extension (paid with a 30-day free trial). It turns your editor into a powerful environment designed specifically for React Native and Expo projects, with advanced debugging, a network inspector, router integration, and other built-in tools.
```

I don't think there's a need to describe 30-day trial (things like these get outdated quite soon from the main marketing website) and could be coupled with context when mentioning paid.

Reply:
Yeah, I agree